### ISSUE TITLE
Accessibility: Home page call-to-action "Learn More" could be improved

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#368]
The link text "Learn More" in the initial call-to-action on the [Centers of Excellence home page](https://coe.gsa.gov/) could be improved, because it only makes sense in context of the containing hero callout, which is notably also not part of a single continuous paragraph or sentence.

Note that this may not be a violation of WCAG Level AA ([Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)), but would be a failure of WCAG Level AAA ([Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only)). For the purpose of Section 508 compliance, the latter does not strictly apply (refer to [Revised 508 Standards, E205 Electronic Content](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/text-of-the-standards-and-guidelines)). However, as noted in the failure linked below, the fact that the preceding text "Accelerating modernization across government" is not part of the same paragraph may make it difficult for the user to easily discover to what the link will relate.

**WCAG Success Criteria (AAA):** [2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only)

**Relevant Guidance:**

https://webaim.org/techniques/hypertext/link_text

>Links are more useful when they make sense out of context. Authors should avoid non-informative link phrases such as: [...] read more

https://www.nngroup.com/articles/learn-more-links/

>the lack of descriptive keywords is a serious problem for accessibility. People who are visually impaired and using a screen reader or magnifier aren’t able to quickly glance back to the preceding-paragraph text to see what the Learn More link may refer to.

**Relevant failure:** [F63: Failure of Success Criterion 2.4.4 due to providing link context only in content that is not related to the link](https://www.w3.org/WAI/WCAG21/Techniques/failures/F63)

>If the context for the link is not provided in one of the following ways: [...] in the same sentence, paragraph, list item, or table cell as the link [...] via a suitable ARIA property such as aria-label or aria-labelledby [...] then the user will not be able to find out where the link is going with any ease.

**Remediation Guidance:**

The link text (or equivalent ARIA labelling) should describe where the link will navigate.

**Comment by iamjolly** [comment]
Suggest changing the homepage hero CTA text to "Learn more about COEs" @daniellegallant
